packages/sources/kaiko/README.md

```diff
@@ -2,9 +2,10 @@
 
 ### Environment Variables
 
-| Required? |  Name   |                                 Description                                 | Options | Defaults to |
-| :-------: | :-----: | :-------------------------------------------------------------------------: | :-----: | :---------: |
-|    ✅     | API_KEY | An API key that can be obtained from [here](https://www.coinapi.io/pricing) |         |             |
+| Required? |       Name        |                     Description                      | Options | Defaults to |
+| :-------: | :---------------: | :--------------------------------------------------: | :-----: | :---------: |
+|    ✅     |      API_KEY      |      An API key that can be obtained from Kaiko      |         |             |
+|    ✅     | WS_CONNECTION_KEY | An WS connection key that can be obtained from Kaiko |         |             |
```

Comment:
`WS_CONNECTION_KEY` is not necessary. The `API_KEY` is the same for WS and HTTP

Reply:
Forgot to take this one out, thanks for catching that.